### PR TITLE
adding pristine Zen to tidying branch

### PR DESCRIPTION
**Pull request by tina-trenkner**
I indented a lot of divs and uls. Tbh, I wouldn't pay myself to clean up code at this point.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            </div>
+
+            <div class="zen-resources" id="zen-resources">
+            <h3 class="resources">Resources:</h3>
+                <ul>
+                    <li class="view-css">
+                        <a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">View This Design&#8217;s <abbr title="Cascading Style Sheets">CSS</abbr></a>
+                    </li>
+                    <li class="css-resources">
+                         <a href="http://www.mezzoblue.com/zengarden/resources/" title="Links to great sites with information on using CSS."><abbr title="Cascading Style Sheets">CSS</abbr> Resources</a>
```
Comment (JaivePNG) [inline pristine-zen-garden.html]: Inline elements, put these on same line.

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        </p>
+        <p>
+            We ask that you submit original artwork. Please respect copyright laws. Please keep objectionable material to a minimum, and try to incorporate unique and interesting visual themes to your work. We&#8217;re well past the point of needing another garden-related design.
+        </p>
+        <p>
+            This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.
+        </p>
+        <p role="contentinfo">
```
Comment (JaivePNG) [inline pristine-zen-garden.html]: Inline elements: you can put these on the same line

**Comment by JaivePNG** [comment]
Hi Tina, please check the following:
1. Inline Elements indentation: line 93 -94, line 163 - 164 ( also have a look at the rest on line 151 - 171).
2. Also check your <p> indentation in between the <div class = benefits etc etc> on line 72 to the closing </div> on line 96.
3. Overall, yours is much easier to read to than mine. 
Once yo have done this, let me know and I can close this for you.

**Comment by JaivePNG** [comment]
@tina-trenkner Hi Tina, when you done, let me know so I can review and accept your pull request. and then delete this branch.

**Comment by tina-trenkner** [comment]
Hi @JaivePNG, thanks for taking a look. I get out of work at 3, so I think the earliest I can make changes is around 5. Could I ping you when I've made the changes? Happy to tell Rich and Tony that the delay is on me. -T

**Comment by JaivePNG** [comment]
@tina-trenkner sorry Tina, no delay - its cool whenever you are ready. I thought once we are both done, we can close this activity off. So was just letting you know. its cool. Ping me when you ready.

**Comment by tina-trenkner** [comment]
@JaivePNG Hey sir, thanks for being patient with me. I think I made the suggestions you've laid out. Whatddyathink?

**Comment by JaivePNG** [comment]
Hi Tina, your work is very neat. I am closing this now.

**Pull request merged**